doc/man/salt.7

```diff
@@ -383685,7 +383685,7 @@ bond0:
     \- max_bonds: 1
     \- updelay: 0
     \- use_carrier: on
-    \- xmit_hash_policy: layer2
+    \- hashing-algorithm: layer2
```

Comment:
this file is generated when we build the man pages for each release, so we should never directly edit this file

Reply:
ping @waynew looks like you made this change originally, mind taking a look here? Or i can make the change